en/micro-integrator/docs/references/connectors/servicenow-connector/servicenow-connector-example.md

```diff
@@ -0,0 +1,250 @@
+# ServiceNow Connector Example
+
+The ServiceNow connector allows you to access the ServiceNow REST API through WSO2 EI. ServiceNow is a software platform that supports IT service management and automates common business processes. This software as a service (SaaS) platform contains a number of modular applications that can vary by instance and user. 
+
+It focuses on service-orientation toward the tasks, activities, and processes.
```

Comment:
The introduction of what is `serviceNow` does not give a overall idea what it is and what it does. 

1. This video has what it can do. https://www.youtube.com/watch?v=g1g9iJc6WyY
2. We also need to say what this connector does. Not only what serviceNow does. The connector can be used to trigger workflows? Or can it create workflows? 
3. Add a link to servicenow site  and the API we support. 
https://developer.servicenow.com/dev.do#!/reference/api/orlando/rest/c_TableAPI

Reply:
Done